addons/website/static/src/xml/website_form_editor.xml

```diff
@@ -289,13 +289,18 @@
             <t t-call="website.form_field">
                 <select class="form-select s_website_form_input" t-att-name="field.name" t-att-required="field.required || field.modelRequired || None" t-att-id="field.id" style="display: none">
                     <t t-foreach="field.records" t-as="record" t-key="record_index">
-                        <option t-esc="record.display_name" t-att-value="record.id" t-att-selected="record.selected and 'selected' or None"/>
+                        <option t-attf-selected="#{record.selected and 'selected' or None}" t-esc="record.display_name" t-att-value="record.id"/>
                     </t>
                 </select>
                 <div id="editable_select" class="form-control s_website_form_input">
                     <t t-foreach="field.records" t-as="record" t-key="record_index">
                         <t t-set="noValueLabel">no value</t>
-                        <div t-esc="record.display_name" t-attf-data-empty-value="&lt;#{noValueLabel}&gt;" t-att-id="record.id" t-attf-class="s_website_form_select_item #{record.selected and 'selected' or ''}"/>
+                        <t t-if="record.selected">
+                            <div class="d-flex justify-content-between align-items-center">
+                                <div t-esc="record.display_name" t-attf-data-empty-value="&lt;#{noValueLabel}&gt;" t-att-id="record.id" t-attf-class="s_website_form_select_item #{record.selected and 'selected' or ''}"/>
+                                <i class="fa fa-caret-down"/>
+                            </div>
+                        </t>
```

Comment:
You can set the t-if on the div directly no ?

This changes breaks the selection field :grimacing: 
![image](https://github.com/odoo/odoo/assets/78849981/21facda6-d19c-4582-b776-d8690e1036f3)

Reply:
Argh, the selected property is not set by default on the first option as in the existingFields. I'll adjust that.